packages/cli/src/service/templates/client.tpl.ts

```diff
@@ -18,13 +20,11 @@ export type {
   ${upperName}Data,
   ${upperName}Query
 }
-`
-
-const methodsTemplate = ({ camelName, upperName, className, type }: ServiceGeneratorContext) => `
-const ${camelName}ServiceMethods = ['find', 'get', 'create', 'update', 'patch', 'remove'] as const
-type ${upperName}ClientService = Pick<${className}${
+export const ${camelName}ServiceMethods = ['find', 'get', 'create', 'patch', 'remove'] as const
```

Comment:
Missing `update`?
Noticed in a newly generated app.

Reply:
You can add it if you are using it but we found that `patch` in most cases provides the exact same functionality, at least for the database adapters.

Reply:
Totally understandable, makes sense and makes things easier 😃